Tests build failure on aarch64-linux

### Checklist

- [x] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT/wiki/Hardware-Support) as a known limitation. Examples of known driver-side issues are fans not turning on with a custom curve on RDNA3/4 and missing fan control on laptops.
- [x] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration
- [x] I've checked that the issue still happens in the latest [development build](https://github.com/ilya-zlobintsev/lact?tab=readme-ov-file#development-builds).

### Bug description

As we discovered in https://github.com/NixOS/nixpkgs/pull/411886#issuecomment-2917627515, building on aarch64 is broken.

<details>
<summary>Error logs: `aarch64-linux</summary>
<pre>   Compiling easy_fuser v0.4.0
error[E0308]: mismatched types
   --> /build/lact-0.7.4-vendor/easy_fuser-0.4.0/src/unix_fs.rs:844:56
    |
844 |         unsafe { unix_impl::listxattr(c_path.as_ptr(), buf.as_mut_ptr() as *mut i8, buf.len()) };
    |                  --------------------                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*mut u8`, found `*mut i8`
    |                  |
    |                  arguments to this function are incorrect
    |
    = note: expected raw pointer `*mut u8`
               found raw pointer `*mut i8`
note: function defined here
   --> /build/lact-0.7.4-vendor/easy_fuser-0.4.0/src/unix_fs/linux_fs.rs:58:22
    |
58  | pub(super) unsafe fn listxattr(path: *const c_char, list: *mut c_char, size: size_t) -> ssize_t {
    |                      ^^^^^^^^^                      -----------------

For more information about this error, try `rustc --explain E0308`.
error: could not compile `easy_fuser` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...</pre>
</details>

### System info

```markdown
- LACT version:0.7.4
- GPU model: 6700xt
- Kernel version: 6.15
- Distribution: NixOS
```

--- a/flatpak/generated-sources.json
+++ b/flatpak/generated-sources.json
@@ -779,14 +779,14 @@
     {
         "type": "archive",
         "archive-type": "tar-gzip",
-        "url": "https://static.crates.io/crates/easy_fuser/easy_fuser-0.4.0.crate",
-        "sha256": "7f3ed26d190ef40be51fd728975117f58a30df5a83da7af5d52b98a92277d176",
-        "dest": "cargo/vendor/easy_fuser-0.4.0"
+        "url": "https://static.crates.io/crates/easy_fuser/easy_fuser-0.4.1.crate",
+        "sha256": "741f7677b0bf516435724bc9be16b5f14ec925e8dffd5059bc8d7ca006b00eeb",
+        "dest": "cargo/vendor/easy_fuser-0.4.1"
     },
     {
         "type": "inline",
-        "contents": "{\"package\": \"7f3ed26d190ef40be51fd728975117f58a30df5a83da7af5d52b98a92277d176\", \"files\": {}}",
-        "dest": "cargo/vendor/easy_fuser-0.4.0",
+        "contents": "{\"package\": \"741f7677b0bf516435724bc9be16b5f14ec925e8dffd5059bc8d7ca006b00eeb\", \"files\": {}}",
+        "dest": "cargo/vendor/easy_fuser-0.4.1",
         "dest-filename": ".cargo-checksum.json"
     },
     {

--- a/lact-daemon/Cargo.toml
+++ b/lact-daemon/Cargo.toml
@@ -53,7 +53,7 @@ cl3 = { git = "https://github.com/kenba/cl3", branch = "develop", features = [
 [dev-dependencies]
 pretty_assertions = { workspace = true }
 insta = { version = "1.41.1", features = ["json", "yaml"] }
-easy_fuser = { version = "0.4.0", features = ["parallel"] }
+easy_fuser = { version = "0.4.1", features = ["parallel"] }
 tempfile = "3.17.1"
 
 [build-dependencies]

